docs/content_credentials.md

```diff
@@ -0,0 +1,96 @@
+# Content Credential Proposal
+
+## Background
+We have long struggled with the need to save and later continue to edit C2PA manifests.
+The spec doesn't call out a specific way to do this. It only allows for appending new signed manifests.
+But there are many scenarios where we need a work in progress.
+- Validating an ingredient once and saving it for later use.
+- Frequent small edits over time without creating a long chain of signed manifests
+- Applications that want to keep track of changes over time and export later.
```

Comment:
That I think relates to the first point, so I would move this one to second position